core/transaction-pool/graph/src/pool.rs

```diff
@@ -305,7 +305,6 @@ impl<B: ChainApi> Pool<B> {
 
 impl<B: ChainApi> Pool<B> {
 	/// Create a new transaction pool.
-	/// TODO [ToDr] Options
```

Comment:
This one is actually still valid, logged here: #1579

Reply:
Thanks for logging, @tomusdrw but do you think this _needs_ a code comment here as well? I think the ticket is more than fine enough :) .

Reply:
Yup, I'm actually taking it right away anyway :)